doc/reference/pipeline_spec.md

```diff
@@ -35,7 +35,15 @@ create-pipeline](../pachctl/pachctl_create-pipeline.html) doc.
     "cpu": double
   },
   "input": {
```

Comment:
Same comment: you can't actually have more than one input types at the top level.  Has to be one of `atom/union/cross`.